lntest/node/watcher.go

```diff
@@ -681,6 +681,18 @@ func CheckChannelPolicy(policy, expectedPolicy *lnrpc.RoutingPolicy) error {
 		return fmt.Errorf("expected max htlc %v, got %v",
 			expectedPolicy.MaxHtlcMsat, policy.MaxHtlcMsat)
 	}
+	if policy.InboundFeeBaseMsat != expectedPolicy.InboundFeeBaseMsat {
+		return fmt.Errorf("expected max htlc %v, got %v",
+			expectedPolicy.InboundFeeBaseMsat,
+			policy.InboundFeeBaseMsat)
+	}
+	if policy.InboundFeeRateMilliMsat !=
+		expectedPolicy.InboundFeeRateMilliMsat {
+
+		return fmt.Errorf("expected max htlc %v, got %v",
```

Comment:
nit: error message should refer to inbound fees

Reply:
should be fixed now